edx_rest_framework_extensions/tests/test_jwt_decoder.py

```diff
@@ -149,24 +160,24 @@ def test_decode_failure_invalid_token(self):
             patched_log.error.assert_any_call(msg)
 
     @override_settings(JWT_AUTH=exclude_from_jwt_auth_setting('JWT_SUPPORTED_VERSION'))
-    def test_decode_supported_jwt_version_not_specified(self):
+    def test_supported_jwt_version_not_specified(self):
         """
         Verifies the JWT is decoded successfully when the JWT_SUPPORTED_VERSION setting is not specified.
         """
         token = generate_jwt_token(self.payload)
         self.assertDictEqual(jwt_decode_handler(token), self.payload)
 
     @ddt.data(None, '0.5.0', '1.0.0', '1.0.5', '1.5.0', '1.5.5')
-    def test_decode_supported_jwt_version(self, jwt_version):
+    def test_supported_jwt_version(self, jwt_version):
         """
-        Verifies the JWT is decoded successfully when the JWT_SUPPORTED_VERSION setting is not specified.
+        Verifies the JWT is decoded successfully with different supported versions in the token.
         """
         jwt_payload = generate_jwt_payload(self.user, version=jwt_version)
         token = generate_jwt_token(jwt_payload)
         self.assertDictEqual(jwt_decode_handler(token), jwt_payload)
 
     @override_settings(JWT_AUTH=update_jwt_auth_setting({'JWT_SUPPORTED_VERSION': '0.5.0'}))
-    def test_decode_unsupported_jwt_version(self):
+    def test_unsupported_jwt_version(self):
```

Comment:
This test is doubling as both a test of unsupported version and a test for overriding version using the setting.  Can the name be updated, or can the docstring mention this?